libraries/rush-lib/src/logic/buildCache/AdditionalFilesHasher.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+import { Async, Import, LegacyAdapters } from '@rushstack/node-core-library';
+import { getGitHashForFiles } from '@rushstack/package-deps-hash';
+
+import type { IOptions } from 'glob';
+
+const glob: typeof import('glob') = Import.lazy('glob', require);
+
+const globAsync = (pattern: string, options: IOptions = {}): Promise<string[]> => {
+  return LegacyAdapters.convertCallbackToPromise(glob, pattern, options);
+};
+
+export class AdditionalFilesHasher {
+  public static async getFileHashedFilesMap(
```

Comment:
Since there's only one exported function, this can be cleaner as
`export async function getHashesForGlobsAsync(globPatterns: Iterable<string>, packagePath: string): Promise<Map<string, string>>`
and
`async function expandGlobPatternsAsync(globPatterns: Iterable<string>, packagePath: string): Promise<string[]>`

We have a convention in the rushstack repository of suffixing function names that return promises with `Async`

Reply:
https://github.com/microsoft/rushstack/pull/3824/commits/c2fc78c917ff50fa8a9390e7ace0e25f591ed313

Here `AdditionalFilesHasher.ts` and `getHashesForGlobsAsync.ts` are nearly identical files, but because of indentation more than 30% of lines were changed between these two files and now it seems that one file was deleted one added.